lib/definitions/server-config-manager.d.ts

```diff
@@ -23,16 +23,18 @@ interface IServerConfig {
 	domainName: string;
 	serverProto?: string;
 	stage?: string;
+	cloudEnv?: IStringDictionary;
 	cloudServices: IDictionary<ICloudServiceConfig>;
-	[index: string]: string | IDictionary<ICloudServiceConfig>;
+	[index: string]: any;
```

Comment:
You can extract this as a separate interface named something like `IDynamicIndex` and thus reuse it here and on the line below